examples/endpoints.rs

```diff
@@ -12,6 +12,14 @@ async fn main() -> Fallible<()> {
 
     let bn = Binance::with_credential(&api_key, &secret_key);
 
+    // get aggregated historical trades
+    match bn.get_historical_trades(
+        "BTCUSDT",
+        10, 963563573)?.await {
+        Ok(answer) => println!("{:?}", answer),
+        Err(e) => println!("Error: {}", e),
+    }
```

Comment:
@Areredify could u please give me a hand with this? It's my first time on rust! This new endpoint I've plumbed in fails:

`Error: error decoding response body: data did not match any variant of untagged enum BinanceResponse`  which I'm guessing is something to do with deserialization of the response here? https://github.com/1e9-io/binance-async-rs/blob/d70c34c094ebce8925457a594f28ead7b0b69f3a/src/error.rs#L27-L30

I've debugged this with a real API key and can confirm it's making the correct request to binance. Something like:
curl -H "X-MBX-APIKEY: ***redacted***" "https://www.binance.com/api/v3/historicalTrades?fromId=963563573&limit=10&symbol=BTCUSDT" 

Assuming u have an API key and want to see the data for yourself. Thanks!

Reply:
response is something like this

```
[
{"id":963563573,"price":"32079.87000000","qty":"0.08665400","quoteQty":"2779.84905498","time":1626945423535,"isBuyerMaker":false,"isBestMatch":true},
...]
```

Reply:
I think it's not matching because the response returns an array of {} ... guess I gotta do a parent level collection or something, maybe something like https://github.com/1e9-io/binance-async-rs/blob/e5e5d8ecde5a6a1e7f49bc718541e57899cc425d/src/client/market.rs#L145-L150